test/integration/lib/expression.js

```diff
@@ -70,19 +70,35 @@ exports.run = function (implementation, options, runExpressionTest) {
 
             if (process.env.UPDATE) {
                 fixture.expected = result;
+                // We save the expected serialization, but the round-tripped compilation results
```

Comment:
This would strip `expected.serialized` if we ran the update from GL JS.  Since we don't have `Expression.serialize()` here, maybe something like:

```js
  fixture.expected = Object.assign({
    // if we're updating from GL JS, where `result.serialized` will not exist, just copy the existing expected value, or default to expected the serialized form to be identical to the input expression
    serialized: (fixture.expected && fixture.expected.serialized) ? fixture.expected.serialized : fixture.expression
  }, result);
```

Reply:
Of course! Good catch...